mobile/lib/models/enum_constants.dart

```diff
@@ -183,59 +182,6 @@ enum AppNotificationType {
   welcomeMessage,
 }
 
-@HiveType(typeId: 140, adapterName: 'RegionAdapter')
-enum Region {
-  @HiveField(1)
-  central('Central Region'),
-  @HiveField(2)
-  eastern('Eastern Region'),
-  @HiveField(3)
-  northern('Northern Region'),
-  @HiveField(4)
-  western('Western Region'),
-  @HiveField(5)
-  southern('Southern Region'),
-  @HiveField(0)
-  none('');
```

Comment:
Thanks @NoahNsimbe 

Great decision, we need to limit our scope when it comes to matters of search. The topic of LOCATION SEARCH needs to be approached with caution since there is no documented perfect solution/standard that can be applied as we expand internationally.

In summary, I foresee us just going with the simplest and scalable approaches for location searches.